examples/ompls/sci.yml

```diff
@@ -30,7 +31,5 @@ graph:
           operational-carbon: 5
           embodied-carbon: 0.02
           requests: 100
-          functional-unit-duration: 1
```

Comment:
Here is exactly an idea of the change, to remove config props pollution on outputs.